Services/UI/OJS.Services.Ui.Business/Implementations/ParticipantsBusinessService.cs

```diff
@@ -34,6 +35,18 @@ public class ParticipantsBusinessService : IParticipantsBusinessService
         this.datesService = datesService;
     }
 
+    public Task<ContestParticipantsViewModel> GetPracticeParticipantsCount(int contestId)
+        => Task.FromResult(new ContestParticipantsViewModel
+        {
+            PracticeParticipantsCount = this.participantsData.GetAllByContestAndIsOfficial(contestId, false).Count(),
```

Comment:
Can you use `CountAsync` and return the task from it

Reply:
Done.